api/app/signals/apps/api/tests/test_reaction_request_flow_trigger.py

```diff
@@ -18,10 +18,8 @@ def setUp(self):
         self.signal = SignalFactory.create(status__state=workflow.GEMELD)
         self.client.force_authenticate(user=self.superuser)
 
-    def test_email_known_REACTIE_GEVRAAGD_state_allowed(self):
-        url = self.detail_endpoint.format(pk=self.signal.id)
         payload = {'status': {'state': workflow.REACTIE_GEVRAAGD, 'text': 'Our question.'}}
-        response = self.client.patch(url, data=payload, format='json')
+        response = self.client.patch(f'/signals/v1/private/signals/{self.signal.pk}', data=payload, format='json')
```

Comment:
Not a blocker at all; why do this? (Our test suite generally uses the `self.detail_endpoint.format(...)` style widely, this seems to break with that.)

Reply:
We use both. Perhaps we should take a look at this in a seperate PR and decide using one or the other.